crates/oct-cloud/src/aws.rs

```diff
@@ -396,9 +356,55 @@ impl Resource for Ec2Instance {
             .ok_or("No instances returned")?;
 
         self.id = instance.instance_id.clone();
-        self.arn = instance.outpost_arn.clone();
-        self.public_ip = instance.public_ip_address.clone();
-        self.public_dns = instance.public_dns_name.clone();
+
+        log::info!("Instance launched, waiting for metadata to be available");
+
+        tokio::time::sleep(std::time::Duration::from_secs(30)).await;
```

Comment:
Do we really need this sleep if we already have enough number of attempts getting the metadata?